mcux/mcux-sdk/drivers/netc/socs/imx943/fsl_netc_soc.c

```diff
@@ -0,0 +1,55 @@
+/*
+ * Copyright 2024 NXP
+ *
+ * SPDX-License-Identifier: BSD-3-Clause
```

Comment:
remove all code for imx943?

Reply:
> remove all code for imx943?

Okay. Updated the pull request and removed.
Thanks.